MAUIsland/Features/Gallery/MAUI/GridLayout/Pages/GridLayoutPage.xaml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<app:BasePage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:app="clr-namespace:MAUIsland"
+             
+             x:Class="MAUIsland.GridLayoutPage"
+             x:DataType="app:GridLayoutPageViewModel"
+             
+             
+             Title="GridLayoutPage">
```

Comment:
Need page resource and all the other parts 
+ ControlInfo
+ List header
+ List item template
+ List footer